source/content/drush-versions.md

```diff
@@ -6,7 +6,9 @@ categories: [drupal,workflow,platform,develop]
 ---
 By default, Pantheon runs Drush 8 on newly created Drupal sites. Drush 8 is compatible with Drupal 6, 7, and 8.
 
-The Drush version is set when the site is created. Sites created before Nov 4, 2015 (and Drupal 8 sites created before Sep 3, 2015) use an older version of Drush that is incompatible with later versions of PHP. You can [check your current Drush version via Terminus](/drush-versions/#verify-current-drush-version). Updating Drush can be accomplished by [a simple addition to your pantheon.yml file](/drush-versions/#configure-drush-version) below.
+The Drush version is set when the site is created. Sites created before Nov 4, 2015 (and Drupal 8 sites created before Sep 3, 2015) use an older version of Drush that is incompatible with later versions of PHP. You can [check your current Drush version via Terminus](#verify-current-drush-version). Updating Drush can be accomplished by [a simple addition to your pantheon.yml file](#configure-drush-version) below.
```

Comment:
Do we still want to hilite the Drush version selection rules from 2015? If this info is relevant at all, maybe it should be a footnote somewhere. We never put the Drush version in pantheon.upstream.yml, but I still think we could simplify here and take out the 2015's.

"The Drush version is set to the version of Drush that was preferred at the time the site was created. The preferred version of Drush is currently 8."